From 62ec63fab15b1835c94212f47c04636662e0ac76 Mon Sep 17 00:00:00 2001
From: George Hopkins <george-hopkins@null.net>
Date: Thu, 31 May 2018 12:55:06 +0200
Subject: [PATCH 1/2] Disable fatal linker warnings

libQt5Network.so depends on EVP_CipherFinal which throws a warning.

Signed-off-by: George Hopkins <george-hopkins@null.net>
---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cb35545..357fafe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -143,6 +143,9 @@ if (EXISTS "${CMAKE_SOURCE_DIR}/.git")
   endif()
 endif()
 
+set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--no-fatal-warnings")
+set(CMAKE_MODULE_LINKER_FLAGS "${CMAKE_MODULE_LINKER_FLAGS} -Wl,--no-fatal-warnings")
+
 configure_file(config.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/src/config.h)
 
 # Include dirs used everywhere
-- 
2.7.4

